Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -1,32 +1,68 @@
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Runtime.Serialization;
 using Microsoft.EntityFrameworkCore;
 
 namespace Corgibytes.Freshli.Cli.Functionality
 {
+    [Serializable]
+    public class CacheException : Exception
+    {
+        public bool IsWarning { get; init; }
```

Comment:
I didn't have a chance to chat with you about this approach before I committed it. I'm curious what you think about it.

Reply:
Frankly, I think it detracts from readability, but no sense taking additional time to fix it now.